File: test/data/usecase/src/cs/ShootNumber.cs

int OldCanopyShootNumber = CanopyShootNumber;
EmergedLeaves = (int)Math.Max(1, Math.Ceiling(LeafNumber - 1));
Shoots = fibonacci(EmergedLeaves);

CanopyShootNumber = Math.Min(Shoots * SowingDensity, TargetFertileShoot);
AverageShootNumberPerPlant = CanopyShootNumber / SowingDensity;

if (CanopyShootNumber != OldCanopyShootNumber)
{
	tilleringProfile.Add(CanopyShootNumber - OldCanopyShootNumber);
}
		
TillerNumber = tilleringProfile.Count;
		
for (int i = leafTillerNumberArray.Count; i < Math.Ceiling(LeafNumber); i++)
{
	leafTillerNumberArray.Add(TillerNumber);
}

File: test/data/usecase/src/py/ShootNumber.py
OldCanopyShootNumber = CanopyShootNumber
EmergedLeaves = int(max(1, ceil(LeafNumber - 1)))

Shoots = fibonacci(EmergedLeaves)

CanopyShootNumber = min(Shoots * SowingDensity, TargetFertileShoot)
AverageShootNumberPerPlant = CanopyShootNumber / SowingDensity
 
        
if (CanopyShootNumber != OldCanopyShootNumber):
    diff = CanopyShootNumber - OldCanopyShootNumber
    tilleringProfile.append(diff)
           
TillerNumber = len(tilleringProfile)
        
for i in range(len(leafTillerNumberArray),int(ceil(LeafNumber))):
    leafTillerNumberArray.append(TillerNumber)
    



